_events/2022-07-22-10 year anniversary.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 10 Year anniversary 🎉
```

Comment:
Set permalink to `events/10YearAnniversary` to keep links working?

Reply:
Meh, I would rather keep the file structure clean - happy to accept a one off article or toot with a broken link. That URL was only to download the ICS anyway.

Reply:
I find it a bit sad to possibly break links, but okay